From 3d996e218cf24b85a4637d90bd0aeac6a95cd9cf Mon Sep 17 00:00:00 2001
From: Lukas Slebodnik <lslebodn@redhat.com>
Date: Sat, 5 Mar 2016 15:15:20 +0100
Subject: [PATCH 100/108] UTIL: Fix indentation in dlinklist.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
Reviewed-by: Michal Židek <mzidek@redhat.com>
(cherry picked from commit fee2997ff25938bca8dd8e3df1d6a5a44b5b7698)
(cherry picked from commit 52d29d5778114afb605fca36efeba2c99f3bff5f)
---
 src/util/dlinklist.h | 130 +++++++++++++++++++++++++--------------------------
 1 file changed, 65 insertions(+), 65 deletions(-)

diff --git a/src/util/dlinklist.h b/src/util/dlinklist.h
index ef09661f4be4d6e973173887f7790f214bf0f1dd..e8490496c4d8a8018edbeec776b2f09d2066236c 100644
--- a/src/util/dlinklist.h
+++ b/src/util/dlinklist.h
@@ -27,112 +27,112 @@
 /* hook into the front of the list */
 #define DLIST_ADD(list, p) \
 do { \
-        if (!(list)) { \
-		(list) = (p); \
-		(p)->next = (p)->prev = NULL; \
-	} else { \
-		(list)->prev = (p); \
-		(p)->next = (list); \
-		(p)->prev = NULL; \
-		(list) = (p); \
-	}\
+    if (!(list)) { \
+        (list) = (p); \
+        (p)->next = (p)->prev = NULL; \
+    } else { \
+        (list)->prev = (p); \
+        (p)->next = (list); \
+        (p)->prev = NULL; \
+        (list) = (p); \
+    } \
 } while (0)
 
 /* remove an element from a list - element doesn't have to be in list. */
 #define DLIST_REMOVE(list, p) \
 do { \
-	if ((p) == (list)) { \
-		(list) = (p)->next; \
-		if (list) (list)->prev = NULL; \
-	} else { \
-		if ((p)->prev) (p)->prev->next = (p)->next; \
-		if ((p)->next) (p)->next->prev = (p)->prev; \
-	} \
-	if ((p) != (list)) (p)->next = (p)->prev = NULL; \
+    if ((p) == (list)) { \
+        (list) = (p)->next; \
+        if (list) (list)->prev = NULL; \
+    } else { \
+        if ((p)->prev) (p)->prev->next = (p)->next; \
+        if ((p)->next) (p)->next->prev = (p)->prev; \
+    } \
+    if ((p) != (list)) (p)->next = (p)->prev = NULL; \
 } while (0)
 
 /* promote an element to the top of the list */
 #define DLIST_PROMOTE(list, p) \
 do { \
-          DLIST_REMOVE(list, p); \
-          DLIST_ADD(list, p); \
+    DLIST_REMOVE(list, p); \
+    DLIST_ADD(list, p); \
 } while (0)
 
 /* hook into the end of the list - needs a tmp pointer */
 #define DLIST_ADD_END(list, p, type) \
 do { \
-		if (!(list)) { \
-			(list) = (p); \
-			(p)->next = (p)->prev = NULL; \
-		} else { \
-			type tmp; \
-			for (tmp = (list); tmp->next; tmp = tmp->next) { \
-				/* no op */ \
-			} \
-			tmp->next = (p); \
-			(p)->next = NULL; \
-			(p)->prev = tmp; \
-		} \
+    if (!(list)) { \
+        (list) = (p); \
+        (p)->next = (p)->prev = NULL; \
+    } else { \
+        type tmp; \
+        for (tmp = (list); tmp->next; tmp = tmp->next) { \
+            /* no op */ \
+        } \
+        tmp->next = (p); \
+        (p)->next = NULL; \
+        (p)->prev = tmp; \
+    } \
 } while (0)
 
 /* insert 'p' after the given element 'el' in a list. If el is NULL then
    this is the same as a DLIST_ADD() */
 #define DLIST_ADD_AFTER(list, p, el) \
 do { \
-        if (!(list) || !(el)) { \
-		DLIST_ADD(list, p); \
-	} else { \
-		p->prev = el; \
-		p->next = el->next; \
-		el->next = p; \
-		if (p->next) p->next->prev = p; \
-	}\
+    if (!(list) || !(el)) { \
+        DLIST_ADD(list, p); \
+    } else { \
+        p->prev = el; \
+        p->next = el->next; \
+        el->next = p; \
+        if (p->next) p->next->prev = p; \
+    } \
 } while (0)
 
 /* demote an element to the end of the list, needs a tmp pointer */
 #define DLIST_DEMOTE(list, p, type) \
 do { \
-		DLIST_REMOVE(list, p); \
-		DLIST_ADD_END(list, p, type); \
+    DLIST_REMOVE(list, p); \
+    DLIST_ADD_END(list, p, type); \
 } while (0)
 
 /* concatenate two lists - putting all elements of the 2nd list at the
    end of the first list */
 #define DLIST_CONCATENATE(list1, list2, type) \
 do { \
-		if (!(list1)) { \
-			(list1) = (list2); \
-		} else { \
-			type tmp; \
-			for (tmp = (list1); tmp->next; tmp = tmp->next) { \
-				/* no op */ \
-			} \
-			tmp->next = (list2); \
-			if (list2) { \
-				(list2)->prev = tmp;	\
-			} \
-		} \
+    if (!(list1)) { \
+        (list1) = (list2); \
+    } else { \
+        type tmp; \
+        for (tmp = (list1); tmp->next; tmp = tmp->next) { \
+            /* no op */ \
+        } \
+        tmp->next = (list2); \
+        if (list2) { \
+            (list2)->prev = tmp; \
+        } \
+    } \
 } while (0)
 
 /* insert all elements from list2 after the given element 'el' in the
  * first list */
 #define DLIST_ADD_LIST_AFTER(list1, el, list2, type) \
 do { \
-                if (!(list1) || !(el) || !(list2)) { \
-                    DLIST_CONCATENATE(list1, list2, type); \
-                } else { \
-                    type tmp; \
-                    for (tmp = (list2); tmp->next; tmp = tmp->next) { \
-                        /* no op */ \
-                    } \
-                    (list2)->prev = (el); \
-                    tmp->next = (el)->next; \
-                    (el)->next = (list2); \
-                    if (tmp->next != NULL) tmp->next->prev = tmp; \
+    if (!(list1) || !(el) || !(list2)) { \
+        DLIST_CONCATENATE(list1, list2, type); \
+    } else { \
+        type tmp; \
+        for (tmp = (list2); tmp->next; tmp = tmp->next) { \
+            /* no op */ \
+        } \
+        (list2)->prev = (el); \
+        tmp->next = (el)->next; \
+        (el)->next = (list2); \
+        if (tmp->next != NULL) tmp->next->prev = tmp; \
     } \
 } while (0);
 
 #define DLIST_FOR_EACH(p, list) \
-	for ((p) = (list); (p) != NULL; (p) = (p)->next)
+    for ((p) = (list); (p) != NULL; (p) = (p)->next)
 
 #endif /* _DLINKLIST_H */
-- 
2.7.3

